<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Saver_Loader</class>
 <widget class="QWidget" name="Saver_Loader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>xO - Saver</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QGroupBox" name="job_grp">
            <property name="title">
             <string>Job</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="show_lbl">
                   <property name="text">
                    <string>Show:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbb_show"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Shot:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbb_shot"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="task_grp">
            <property name="title">
             <string>Task</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="dep_lbl">
                   <property name="text">
                    <string>Asset:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbb_asset">
                   <item>
                    <property name="text">
                     <string>Character</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Environment</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Prop</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="task_lbl">
                   <property name="text">
                    <string>Task:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbb_task">
                   <item>
                    <property name="text">
                     <string>Modeling</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Rigging</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>LookDev</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Shading</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Compositing</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Prep</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Layout</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Lighting</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Tracking</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>FX</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="btn_refresh">
                 <property name="text">
                  <string>Refresh</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Asset Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_assetname"/>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_active">
            <property name="text">
             <string>Active</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="version_grp">
            <property name="title">
             <string>Version Info</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item alignment="Qt::AlignTop">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Verssions:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QListView" name="list_scripts_versions"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="btn_save">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="lbl_user">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lbl_user_info">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="btn_open">
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
